src/config/chain_config.js

```diff
@@ -7,8 +7,8 @@ class ChainConfig {
    * @param {string} [filePath] An optional config file to overwrite any defaults or env variables.
    */
   constructor(filePath) {
-    this.originWeb3Provider = process.env.ORIGIN_WEB3_PROVIDER || 'http://localhost:8545';
-    this.auxiliaryWeb3Provider = process.env.AUXILIARY_WEB3_PROVIDER || 'http://localhost:8547';
+    this.originWeb3Provider = process.env.ORIGIN_WEB3_PROVIDER || 'http://localhost:8546';
+    this.auxiliaryWeb3Provider = process.env.AUXILIARY_WEB3_PROVIDER || 'http://localhost:8545';
```

Comment:
why change defaults?

Reply:
Fixed 👍  Defaults doesn't needs to be changed.